src/contentmanagerdelegate.cpp

```diff
@@ -172,7 +174,7 @@ void ContentManagerDelegate::paintBookDescription(QPainter *p, const QRect &r, c
     /* Align text to the x coordinate of column index 1, the Name column. */
     nRect.setX(view->visualRect(index.parent().siblingAtColumn(1)).x());
     nRect.setWidth(view->width());
-    p->drawText(nRect, Qt::AlignLeft | Qt::AlignVCenter, index.data(Qt::UserRole+1).toString());
+    p->drawText(nRect, index.data(Qt::UserRole+1).toString(), descTextOp);
```

Comment:
Long text isn't wrapped nicely:

![Screenshot from 2024-06-03 13-49-41](https://github.com/kiwix/kiwix-desktop/assets/11933502/afb4218b-9316-4a6b-8dfc-82a26dd10904)

At the least, I think that the width of the box where the text is drawn should be reduced.

But a much better solution will be if you can completely eliminate the need for custom drawing of the description via `ContentManagerDelegate` and fully rely on Qt's standard handling of tree views. Then, hopefully, the height of the description line (which is now fixed) will automatically adjust to the amount of content in it.

Reply:
Done